models.py

```diff
@@ -0,0 +1,36 @@
+import re
+
+class Phage:
+  supported_databases = {
+    # European Nucleotide Archive phage database
+    "ENA": "^gi\|[0-9]+\|ref\|([^\|]+)\|\ ([^,]+)[^$]*$",
```

Comment:
The regexes should be raw strings; theinvalid escapes will cause warnings in Python 3.6.